README.md

```diff
@@ -1,27 +1,86 @@
-# Forks and Clones
+# Forking and Cloning Github Repositories
+
+## Problem Statement
+
+If you are familiar with how to create local repositories, you have the ability to
+create a logged history of your project. What's great about open source is that lots
+of people are creating new repositories all the time and sharing their work. You can
+take your repository and push it to a remote to share it, or you can clone a repository
+to your local machine from a remote and make a copy on your own system. Due to permissions
+issues you might need to _fork_ a repository before you _clone_ it. How do you clone from
+a remote to your local system and how to do you fork when necessary?
 
 ## Objectives
 
-1. Fork a repo
-2. Clone a repo
+1. Define _remote_
+2. Use `git clone` to copy a repository to your local machine
+3. Use `git remote` to list remotes
+4. Use `git fork` via GitHub to duplicate other organization's repositories into your own
+
+## Define _remote_
+
+To work with or collaborate on any Git project, you need to be able to manage your _remote_
+repositories. Remote repositories are versions of your project that are hosted online, in this
+case, on Github. With these remote repositories, you'll be pushing and pulling data to and from
+them when you need to share work. To work with remotes, you'll get familiar with _fork_ and
```

Comment:
the objective is define remote. Do we need to talk about fork and clone *here* when they live in their own obj. later? I'd cut from To work with...to end of para.